code/game/machinery/kitchen/smartfridge.dm

```diff
@@ -135,6 +143,17 @@
 		return 1
 	return 0
 
+/obj/machinery/smartfridge/secure/chemistry/chemcache
+	name = "\improper Smart Chemistry Chem Cache"
+	desc = "A refrigerated storage unit for holding chemical mixtures for later use."
+	initial_contents = list(

```

Comment:
Ah, well... I tried to make a replacement for the reagents storage locker that would allow for better reagents management even after the supply had been depleted, but I couldn't manage to get it to work - it wouldn't fill with the items on map spawn. So, I've got the vestigial code for it here, even though I don't actually use it.